app/views/project/editor/pdf.pug

```diff
@@ -1,4 +1,5 @@
 div.full-size.pdf(ng-controller="PdfController")
+	.FIXME(ng-controller="AutoCompileController")
```

Comment:
I guess we might need some UI for this at some point that will go here? But if not, this logic probably shouldn't live in a controller

Reply:
A controller is what Brian suggested, but yes I think it's simpler to just pull it into the `PdfController`